lib/install/config/babel.config.js

```diff
@@ -30,6 +30,7 @@ module.exports = function(api) {
         {
           forceAllTransforms: true,
           useBuiltIns: 'entry',
+          corejs: 3,
           modules: false,
           exclude: ['transform-typeof-symbol']
         }
```

Comment:
@somebody32 [This](https://babeljs.io/docs/en/babel-plugin-transform-runtime#corejs) suggests that `@babel/plugin-transform-runtime` (line 56) can specify to use `corejs: 3` as dependency. Were you able to make that work? I'm still seeing errors like these when I make that change:

![Screen Shot 2019-04-18 at 11 11 44 PM](https://user-images.githubusercontent.com/7811733/56375338-680a4800-622f-11e9-9e43-d8216f547f47.png)

Reply:
great point, looks like I need to extend that too (was not using it in our project, that is why didn't spot the problem): https://github.com/zloirock/core-js/blob/master/docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md#babelruntime

tldr: try to add `@babel/runtime-corejs3` package